services/src/main/java/org/exoplatform/webconferencing/notification/provider/WebTemplateProvider.java

```diff
@@ -0,0 +1,67 @@
+package org.exoplatform.webconferencing.notification.provider;
```

Comment:
header is missing

Reply:
It is not the good one. LGPL is for meeds-io organization. For exoplatform org it is AGPL
Please update